VMware.PSDesiredStateConfiguration.md

```diff
@@ -0,0 +1,147 @@
+## Getting Started
+
+**VMware.PSDesiredStateConfiguration** provides commands to compile and execute **DSC Configuration** without using the DSC Local Configuration Manager. Compiled configuration is stored in memory as PS Object. Set, Get, Test-... cmdlets use Invoke-DSCResource cmdlet of PSDesiredStateConfiguration module to execute/run/process the compiled configuration. This solution allows cross platform support on MacOS and Linux. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.
```

Comment:
Provides a set of commands to compile and execute "DSC Configurations"

Compiled DSC Configurations are stored in memory as PSObjects.

The Set, Test and Get-VmwDscConfiguration cmdlets use the **Invoke-DscResource** cmdlet from the **PSDesiredStateConfiguration** module.

to work with the existing **DSC Resources** in the **VMware.vSphereDSC**.